pkgs/development/python-modules/cirq/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, google_api_python_client
+, matplotlib
+, networkx
+, numpy
+, pandas
+, protobuf  # pkgs.protobuf, not pythonPackages.protobuf
+, pythonProtobuf  # pythonPackages.protobuf
+, requests
+, scipy
+, sortedcontainers
+, sympy
+, typing-extensions
+  # test inputs
+, pytestCheckHook
+, pytest-benchmark
+, ply
+, pydot
+, pyyaml
+, pygraphviz
+}:
+
+buildPythonPackage rec {
+  pname = "cirq";
+  version = "0.6.1";
+
+  disabled = pythonOlder "3.5";
+
+  # TODO: replace fetchFromGitHub with fetchPypi in future if don't need patching
+  # Pypi doesn't host source code for Cirq, only wheels, which doesn't work with requirements patch
```

Comment:
Why would we ever want to replace the source with the wheel then?

Reply:
Fair point.